packages/profile-sync-controller/src/controllers/user-storage/schema.test.ts

```diff
@@ -0,0 +1,40 @@
+import { getFeatureAndKeyFromPath, USER_STORAGE_SCHEMA } from './schema';
+
+describe('user-storage/schema.ts', () => {
+  describe('getFeatureAndKeyFromPath', () => {
+    it('should throw error if the feature.key format is incorrect', () => {
+      const path = 'feature/key';
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      expect(() => getFeatureAndKeyFromPath(path as any)).toThrow(
```

Comment:
Yep this is a valid `any`.

Lets make a reusable type above to reduce these eslint comments.

Something like:
```ts
// NOTE - type is used for mocking erroneous values.
// eslint-disable-next-line @typescript-eslint/no-explicit-any
type MockType = any;
```